lib/unauthorized.js

```diff
@@ -15,11 +15,11 @@ module.exports = function(app, config, logger) {
 
 function addWwwAuthenticateHeaders(req, res, domain, clientId, resourceServer, realm) {
   var v1WwwAuthenticateHeader = util.format('Bearer realm="%s", scope="client_id=%s service=%s"',
-    domain, clientId, req.protocol + "://" + req.hostname + req.baseUrl);
+    domain, clientId, "https://" + req.hostname + req.baseUrl);

```

Comment:
Anything coming through api gateway seems to have a protocol of http. I can't think of a case where someone (following best practices) doesn't want their service's url to be https.